app/javascript/controllers/hello_controller.js

```diff
@@ -0,0 +1,7 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
```

Comment:
Are we using this anywhere?

Reply:
I think this is mostly here to provide a quick example if someone wants to make a Stimulus controller. I don't think we expect anyone to use Stimulus in this interview, but wanted to provide a scaffold in case they do.

Reply:
I like it.